### PR TITLE
update readme for kayvee.2

### PR DESCRIPTION
**Pull request by mohit**

### CODE REVIEW

**Comment by mohit** [comment]
cc: @peternga

File: README.md [hunk 1]
```diff
@@ -25,12 +49,60 @@ Run `make test` to execute the tests
...
+
+You can also individually set the `config` using: 
+
+* `setLogLevel`: defaults to `LOG_LEVELS.Debug`
+* `setFormatter`: defaults to `kv.format`
+* `setOutput`: defaults to `console.error`
+
+#### kayvee/logger logging
+
+Unstructured:
```
Comment (nathanleiby) [inline README.md]: Note: These will write structured logs, there just isn't additional metadata

File: README.md [hunk 2]
```diff
@@ -5,7 +5,31 @@ Package kayvee provides methods to output human and machine parseable strings.
...
+```coffee
+log.gaugeD "gauge-title", 3, {user_id: "value", scope: "scope_system"}
+log.gauge "gauge-simple", 18
+```
+
+Or string and structured logs:
+
+```coffee
+log.infoD "non-metric-log", {"msg": "this is my info", user: "user-id", group: "group-id"}
+log.error "really. there is an error with nothing structured about it"
```
Comment (nathanleiby) [inline README.md]: nit: writes a json structured log

**Comment by nathanleiby** [comment]
LGTM barring structured logs

**Comment by mohit** [comment]
updated to clarify `with metadata` as opposed to structured and unstructured.

**Pull request merged**